ospfd/ospf_vty.c

```diff
@@ -139,23 +140,38 @@ ospf_oi_count (struct interface *ifp)
 
 DEFUN_NOSH (router_ospf,
        router_ospf_cmd,
-       "router ospf [(1-65535)]",
+       "router ospf [(1-65535)] [vrf VRFNAME]",
```

Comment:
@eqvinox had me put this back to [vrf NAME] so that we would stay a bit more consistent.  We will still get the vrf auto-complete with the way it was changed